### PR TITLE
Blokku updates

### LINKED ISSUE #26: Synths page display issues
### Description:
Some elements on this page should not get affected by how the user resize the page width

### To Reproduce:
1. Go to https://v2develop.degenerative.finance then click on Explore Synths (if #22 fixed, simply go to https://v2develop.degenerative.finance/synths)
2. Issues list:
- [x] Synths cards sizes are too small to be able to read easily any text, it should just have a fixed width on the space, screenshot 1
- [x] Synths table columns content is affected by the screen width, should also have a fixed width, screenshot 1

### Screenshots:
1. synthsdisplay

### LINKED ISSUE #30: Update Sidebar external links

### LINKED ISSUE #70: Add new theme

### PR DESCRIPTION
**Pull request by blokku-chan**
### Description of changes proposed in this pull request:
- New theme added, updated class names
- Responsive fixes
- Network indicator added
- Menu animation
- CSS purge command added, new purged file included
- Chart height fixes, vertical line & loader added (loader not active, commented out)
- Removed email icon

### Checks:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### CODE REVIEW

File: src/components/Background.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const Background: React.FC = () => {
+  const styles = {
+    zIndex: -1,
+    position: "relative"
+  }
+  
+  return <div style={styles}>
```
Comment (ind-igo) [inline src/components/Background.tsx]: Needs parentheses around the whole return.

File: src/components/layouts/Navbar.tsx [hunk 2]
```diff
@@ -27,8 +28,17 @@ const Navbar = () => {
 
   const toggleDropdown = () => setWalletMenu(!openWalletMenu);
   const toggleMenu = () => setOpenMenu(!openMenu);
-
-  const Navigation: React.FC = () => {
+  
+  let message = "";
+  let messageColor = "";
+  if (chainId != 1) {
+    message = "Wrong network"
```
Comment (ind-igo) [inline src/components/layouts/Navbar.tsx:35]: Will be available for more than just mainnet, like kovan. Will fix this later though.

**Review by ind-igo — APPROVED** [review]

**Comment by ethedev** [comment]
@blokku-chan can you link the issues you worked on and solved, onto this pr? also you took source the master branch you should have taken the `develop` or `degen_development` branch instead, i updated it now, there are some conflicts to be solved before we merge it

@sali0 @blokku-chan the build is failing, @blokku-chan it might just need to resolve the conflicts but i doubt that it would be the only thing

**Comment by ind-igo** [comment]
I was going to fix the build since I am more familiar with the builds.

**Pull request closed without merging**